.github/ISSUE_TEMPLATE/documentation.yml

```
@@ -0,0 +1,155 @@
+name: 📝 Documentation
+description: Report issues with documentation or suggest improvements
+title: "📝 [Docs]: "
+labels: ["documentation", "triage"]
+assignees: []
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for helping improve our documentation! Clear documentation benefits everyone.
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)